### PR TITLE
Fix build version

### PR DESCRIPTION
**Pull request by anilCSE**
---
name: Pull Request
about: Submit a pull request
title: ''
labels: ''
assignees: '@boz @gosuri'

---

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/ovrclk/akash/blob/master/CONTRIBUTING.md))
- [ ] My issue has received approval from the maintainers or lead with the `design/approved` label

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I've read the [CONTRIBUTION](https://github.com/ovrclk/akash/blob/master/CONTRIBUTING.md) guide
- [ ] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,8 +11,8 @@ BUILD_FLAGS = -mod=readonly -tags "netgo ledger" -ldflags \
   -X github.com/cosmos/cosmos-sdk/version.ServerName=akashd \
   -X github.com/cosmos/cosmos-sdk/version.ClientName=akashctl \
   -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger" \
-  -X github.com/cosmos/cosmos-sdk/version.Version=$(shell git rev-parse --abbrev-ref HEAD) \
-  -X github.com/cosmos/cosmos-sdk/version.Commit=$(shell git rev-parse HEAD)'
+  -X github.com/cosmos/cosmos-sdk/version.Version=$(shell echo $(shell git describe --tags) | sed 's/^v//') \
```
Comment (boz) [inline Makefile:14]: just use `$(shell git describe --tags | sed 's/^v//')` (no `echo`)

**Comment by boz** [comment]
What are these build tags actually used for?

I'd like to have one route to creating releases; it's being done with `goreleaser` right now.  Is there a reason why a release binary needs to be built like this outside of `goreleaser`?

**Comment by anilCSE** [comment]
> What are these build tags actually used for?
> 
> I'd like to have one route to creating releases; it's being done with `goreleaser` right now. Is there a reason why a release binary needs to be built like this outside of `goreleaser`?

To check the binary verisons
```
akashd version 
akashd version --long

akashctl version
akashctl version --long
```

**Comment by boz** [comment]
@anilCSE I mean why is it necessary to build the binary like this when `goreleaser` already does it?

**Comment by anilCSE** [comment]
> @anilCSE I mean why is it necessary to build the binary like this when `goreleaser` already does it?

I am not sure if we have used `go-releaser` for previous releases (0.6.0, 0.6.1). When I compile from source, the version is always `HEAD`

**Pull request merged**